### PR TITLE
add example for  kubeadm-userspace,kubeadm-flannel-userspace

### PR DESCRIPTION
**Pull request by hhstu**

### CODE REVIEW

File: manifests/kilo-kubeadm-flannel-userspace.yaml [hunk 1]
```diff
@@ -0,0 +1,176 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kilo
+  namespace: kube-system
+  labels:
+    app.kubernetes.io/name: kilo
+data:
+  cni-conf.json: |
```
Comment (leonnicolas) [inline manifests/kilo-kubeadm-flannel-userspace.yaml:9]: Do we need this with flannel mode? We don't have it in `manifests/kilo-kubeadm-flannel.yaml`

Comment (hhstu) [reply manifests/kilo-kubeadm-flannel-userspace.yaml:9]: yes, i need it

Comment (squat) [reply manifests/kilo-kubeadm-flannel-userspace.yaml:9]: Why is that exactly? Flannel should be providing CNI and all of the CNI binaries so Kilo should be able to be lazy

Comment (hhstu) [reply manifests/kilo-kubeadm-flannel-userspace.yaml:9]: oh, sorry,wo dont`t  need the configmap named kilo when user falannel

Comment (squat) [reply manifests/kilo-kubeadm-flannel-userspace.yaml:9]: Ah ok great! So we can simplify this manifest a bit

Comment (hhstu) [reply manifests/kilo-kubeadm-flannel-userspace.yaml]: done

**Review by leonnicolas — APPROVED** [review]

**Review by squat — APPROVED** [review]

**Pull request merged**